src/components/FeaturePanel/MemberFeatures/MemberFeatures.tsx

```diff
@@ -31,21 +66,21 @@ export const MemberFeatures = () => {
   }
 
   const isClimbingCrag = tags.climbing === 'crag';
-  const heading = isClimbingCrag ? 'Climbing routes' : 'Subitems';
 
   return (
     <Box mb={1}>
       <PanelLabel
         addition={
-          <GradeSystemSelect
-            setGradeSystem={(system) => {
+          <PanelAddition
+            feature={feature}
+            gradeSystem={userSettings['climbing.gradeSystem']}
+            onSetGradeSystem={(system) => {
```

Comment:
Could you please remove all the props and use context inside. As we don't use this component anywhere else, i find it more readable.